build.gradle

```diff
@@ -32,5 +32,40 @@ dependencies {
 }
 
 test {
-	useJUnitPlatform()
+    useJUnitPlatform()
 }
+
+task buildDocker(type: Exec) {
+
+    workingDir "$projectDir/docker-dir/postgres-meta-bug"
+
+    if (System.getProperty('os.name').toLowerCase(Locale.ROOT).contains('windows')) {
+        // No docker for Kris
```

Comment:
Is this going to be always empty? If yes, then I suggest to add negation to the if statement and get rid of "else".

Reply:
Changed